activity/activity.info

```diff
@@ -6,6 +6,7 @@ bundle_id = org.sugarlabs.AbacusActivity
 exec = sugar-activity AbacusActivity.AbacusActivity
 icon = activity-abacus
 show_launcher = yes
-summary = a tool for simple arithmetic calculations
-category = maths
-repository = https://github.com/sugarlabs/abacus
\ No newline at end of file
+website = https://help.sugarlabs.org/en/abacus.html
+summary = A tool for simple arithmetic calculations
+categories = maths
```

Comment:
`categories` is not defined in the metadata reference, please patch `bundle/__init__.py` to add it as an alias for `tags`, and allow either semicolon <kbd>;</kbd> or whitespace delimiters.

Please also add a list of categories to choose from, or a pointer in `bundle/__init__.py` to where the categories are listed in another repository.

In a sample of 78 working activities, 19 use `category` and 8 use `categories`.  None of them used `tags`

Reply:
Before patching the `bundle/__init__.py` 
We can either define right here on what to use and I can make them the same everywhere.  `Categories` or `Category` irrespective of the grammatical part